tools/docker/tests/test_flows.sh

```diff
@@ -664,5 +681,8 @@ main() {
 VERBOSE=false
 SKIP_INIT=false
 STOP_ON_FAILURE=false
+DUMP=false
```

Comment:
I think the default should be `true`. I also think it may be useful to have a separate dump file for each test under tcpdump directory. Then you can get rid of the DUMPFILE parameter (what is it used for?).
This will be extremely helpful IMO to have a separate dump file for each test. But maybe that's just my opinion so (nitpick)

Reply:
Good idea, I can see that being quite useful. It can be difficult to tell based on the capture alone where one test stops and the next one starts, so one file per test sounds like a really good idea.